app/src/components/landing/ruleTemplates/RulePreviewModal/index.js

```diff
@@ -45,9 +46,9 @@ const RulePreviewModal = ({ rule, isOpen, toggle }) => {
       currentOwner,
       modificationDate,
     };
-
+    if (source === "home_screen") trackTemplateImportStarted(snakeCase(ruleToSave.name), source);
```

Comment:
1. change string literal to a constant
2. add comment explaining why we had to add this check